lib/hotel.rb

```diff
@@ -0,0 +1,4 @@
+module Hotel
+
```

Comment:
remove whitespace

Reply:
I'll stop leaving these comments but there's more to remove in other files.

If you `gem install rubocop` and run `rubocop` it will yell at you about style stuff, which is handy.

Reply:
√